frontend/dispatcher.lua

```diff
@@ -623,14 +623,14 @@ function Dispatcher:execute(ui, settings, gesture)
                 local arg = v ~= 0 and v or gesture or 0
                 ui:handleEvent(Event:new(settingsList[k].event, arg))
             end
-            if settingsList[k].configurable ~= nil then
-                 local value = v
-                 if type(v) ~= "number" then
-                     for i, r in ipairs(settingsList[k].args) do
+            if settingsList[k].configurable ~= nil and ui.document then
```

Comment:
Or the other way around?

Reply:
Hmm, arguably, you're less likely to hit a configurable than not being in the Reader, but it's pretty much a coin toss ;).

Reply:
I was talking more or less exclusively about ease of reading, although it could potentially make an impossibly small performance impact one way or the other. ;-)